lib/improver/utilities/cube_manipulation.py

```diff
@@ -832,3 +833,39 @@ def enforce_coordinate_ordering(
     elif anchor == "end":
         cube.transpose(remaining_coords+coord_dims)
     return cube
+
+
+def clip_cube_data(cube, minimum_value, maximum_value):
+    """Apply np.clip to data in a resulting cube to ensure that the limits do
+    not go beyond the limits of the original input cube.
```

Comment:
This needs updating now to limits of the minimum_value and the maximum_value

Reply:
✅